### ISSUE TITLE
Add totals to grids columns

### ISSUE THREAD

**Issue by par456** [APSIMInitiative/ApsimX#9381]
### Describe the new feature

It would be handy if some of the grids showing data had a total field somewhere that added everything up. Some examples include in the chemicals tables to add up all the layers, in the cropsoil for PAWC etc.

**Comment by sno036** [comment]
Agreed. Also in the grids it would be useful to number the rows (1-based probably to match Report indexing). Include LayerStructure in the row numbering!
